qa/L0_backend_python/python_test.py

```diff
@@ -39,15 +39,20 @@
 import tritonclient.http as httpclient
 from tritonclient.utils import *
 
+# By default, find tritonserver on "localhost", but for windows tests
+# we overwrite the IP address with the TRITONSERVER_IPADDR envvar
+_tritonserver_ipaddr = os.environ.get("TRITONSERVER_IPADDR", "localhost")
+
 TEST_JETSON = bool(int(os.environ.get("TEST_JETSON", 0)))
```

Comment:
Is the rationale for this so that we don't confused these variables with constants?

Reply:
It is mainly for consistency in style, for example
```
_tritonserver_ipaddr =     os.environ.get("TRITONSERVER_IPADDR", "localhost")
TEST_JETSON          = ... os.environ.get("TEST_JETSON",         0          ) ...
```
these two are doing almost the exact same thing, so the style in the naming of the variables should be consistent.

You definitely brought up a good point - constants. I wouldn't call those variables constants because they differ every time the script is loaded based on the set environment variables, where constants should be constant. Ideally, I would put those variables inside the class below, for example:
```python
class PythonTest(unittest.TestCase):

  _tritonserver_ipaddr = os.environ.get("TRITONSERVER_IPADDR", "localhost")
  ...

  def setUp(self):
    ...
```
so they become protected variables of the class. If possible, this is even better:
```python
class PythonTest(unittest.TestCase):

  def setUp(self):
    self._tritonserver_ipaddr = os.environ.get("TRITONSERVER_IPADDR", "localhost")
    self._test_jetson = bool(int(os.environ.get("TEST_JETSON", 0)))
    self._test_windows = bool(int(os.environ.get("TEST_WINDOWS", 0)))
    self._shm_leak_detector = shm_util.ShmLeakDetector()
```
https://www.geeksforgeeks.org/protected-variable-in-python/

Reply:
This makes a lot of sense Jacky, thank you for clarifying. Doing a quick search on our repo, it seems like our use of these variables isn't too extensive, so I'll add this task to the folllow-up ticket mentioned below.